OSNetworkRequirementChecker-HLApp/src/ntp-helper.c

```diff
@@ -8,10 +8,10 @@
 #define NTP_SERVER_LEN 256
 
 // List of time server to be tested
-const char *NTPServerList[] = {"13.86.101.172",  "20.43.94.199",   "20.189.79.72",
-                               "40.81.94.65",    "40.81.188.85",   "40.119.6.228",
-                               "51.105.208.173", "51.137.137.111", "51.145.123.29",
-                               "52.148.114.188", "52.231.114.183"};
+const char *NTPServerList[] = { "168.61.215.74", "20.43.94.199", "20.189.79.72",
+                                "40.81.94.65", "40.81.188.85", "40.119.6.228", "40.119.148.38",
```

Comment:
Does the README need a comment, as for the PC version, that 40.81.188.85 may fail if using a commercial ISP in the US?

Reply:
Yes, totally agree, @miz060 I pushed the update in the README.md, if that's ok with you. Thanks.

Reply:
That looks reasonable to me.

Reply:
@gtrevi Yes, that is the case. With the SNTP client fallback mechanism in place, the device should always be able to complete time sync eventually (except when all time server IPs are not reachable).

Reply:
@miz060 Perfect, totally agree and just pushed the mods to disable the fallback (+ consequent README.md update).

Reply:
@gtrevi thanks! the update looks good to me.